little_raft/src/message.rs

```diff
@@ -41,6 +41,7 @@ where
         term: usize,
         success: bool,
         last_index: usize,
+        non_matching_index: Option<usize>,
```

Comment:
How about we call this `mismatch_index` like you call the variables that use it?

Reply:
Fixed